### ISSUE TITLE
If the image title is the same, it doesn't replace the old image when imported.

### ISSUE THREAD

**Issue by mencomcorp** [dweeves/magmi-git#593]
I’m using the Magmi(V0.7.23) for Magento 2 (2.3.4)

If I import an image(ex. MIN-12MP-20-C011.jpg)  to update/replace an existing old image with the same image title (MIN-12MP-20-C011.jpg), the old image is not replaced.

The website uses the cached image and the cached image is not updated even though I perform the flush cache and catalog image.

If the image titles are different, it replaces the old image with no issue.

Where should I look into to resolve the issue?

**Comment by mokanfar** [comment]
I don't think this is a magmi issue because it does what it supposed to do and move image to the media/catalog/product/ folder. the issue is with internals of magento not picking up this change and regenerating. which is understandable. I have just mass appended to file names when replacing existing photos using a program like "batch rename utility"

**Comment by aveevan** [comment]
@mencomcorp May i know how you install Magento into Magento 2.3.4? 

fer download Magmi and paste into usr/shar/nginx/html/magento23/magmi then run url,

http://192.168.1.64:8087/magmi/web/magmi.php

Error : 404 error: Page not found.

**Comment by tmotyl** [comment]
@aveevan @mencomcorp for Magento 2 use this fork https://github.com/macopedia/magmi-m2 (branch magento2)
The one here is dedicated to Magento 1

**Comment by aveevan** [comment]
> @aveevan @mencomcorp for Magento 2 use this fork https://github.com/macopedia/magmi-m2 (branch magento2)
> The one here is dedicated to Magento 1

Thank you.

**Comment by aveevan** [comment]
> @aveevan @mencomcorp for Magento 2 use this fork https://github.com/macopedia/magmi-m2 (branch magento2)
> The one here is dedicated to Magento 1

I am installes same version, after paste magmi folder into magento 2.3 root folder then run 

http://192.168.1.64:8087/magmi/web/magmi.php

Error : 404 error: Page not found.
